### ISSUE TITLE
Running Example.ipynb in binder shows error "libGL.so.1: cannot open shared object file"

### ISSUE THREAD

**Issue by hhoppe** [OpenGeoVis/omfvista#17]
```
File /srv/conda/envs/notebook/lib/python3.10/site-packages/pyvista/plotting/_vtk_gl.py:20
     16 except ImportError:  # pragma: no cover
     17     vtkRenderingContextOpenGL2 = None
---> 20 from vtkmodules.vtkRenderingOpenGL2 import (
     21     vtkCameraPass,
     22     vtkCompositePolyDataMapper2,
     23     vtkDepthOfFieldPass,
     24     vtkEDLShading,
     25     vtkGaussianBlurPass,
     26     vtkOpenGLFXAAPass,
     27     vtkOpenGLHardwareSelector,
     28     vtkOpenGLRenderer,
     29     vtkOpenGLTexture,
     30     vtkRenderPassCollection,
     31     vtkRenderStepsPass,
     32     vtkSequencePass,
     33     vtkShadowMapPass,
     34     vtkSSAAPass,
     35     vtkSSAOPass,
     36 )
     37 from vtkmodules.vtkRenderingVolumeOpenGL2 import (
     38     vtkOpenGLGPUVolumeRayCastMapper,
     39     vtkSmartVolumeMapper,
     40 )

ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

Maybe needs an `apt.txt` file?

**Comment by banesullivan** [comment]
Thanks for pointing this out! I haven't been keeping an eye on this package lately other than the small bit of maintenance I performed recently. I'll try to fix this soon here

**Comment by banesullivan** [comment]
This was fixed by af87100 and 2f40662 although MyBinder is incredibly slow rendering the example these days...
